pallets/keystore/src/benchmarking.rs

```diff
@@ -0,0 +1,26 @@
+use super::*;
+
+#[allow(unused)]
+use crate::Pallet as Identity;
+use codec::alloc::string::{String, ToString};
+use frame_benchmarking::{
+    account as benchmark_account, benchmarks, impl_benchmark_test_suite, whitelisted_caller,
+};
+use frame_support::inherent::Vec;
+use frame_system::RawOrigin;
+
+pub fn from_str_to_vec(string: String) -> Vec<u8> {
+    string.as_bytes().to_vec()
+}
+
+pub fn get_account<T: Config>(name: &'static str) -> T::AccountId {
+    benchmark_account(name, 0, 0)
```

Comment:
Was this from one of the other pallets? We actually shouldn't need this function at all since it just returns the return value of another function.

Reply:
Yup, addressed.